src/createBEMBlock.test.tsx

```diff
@@ -108,4 +109,41 @@ describe('createBEMBlock', () => {
 			<FooBlock block="foo" modifiers="mod2" />,
 		)).toMatchSnapshot()
 	})
+
+	it('Renders README example with correct className as shown', () => {
+		class Bar extends React.Component {
```

Comment:
Whoops! Need to update the README to include `extends React.Component`.

Reply:
i can include the README updates